genologics_sql/queries.py

```diff
@@ -256,7 +256,7 @@ def get_all_samples_in_a_workflow(session, workflow):
                 labworkflow lwf, artifact_sample_map asm, sample sa \
               where st.stageid=stg.stageid and stg.membershipid=wfs.sectionid \
               and wfs.workflowid=lwf.workflowid and wfs.protocolid=lpt.protocolid \
-              and st.artifactid=asm.artifactid and asm.processid=sa.processid and st.completedbyid is null \
+              and st.artifactid=asm.artifactid and asm.processid=sa.processid and st.workflowrunid > 0 \
```

Comment:
Could you explain this one? It feels a bit out of context?

Reply:
Checking `st.workflowrunid > 0` is more correct than checking whether `st.completedbyid is null` which I found out when I was writing the query for `get_currentsteps_protocol_for_sample`